src/App.js

```diff
@@ -3,28 +3,12 @@
 /* eslint-disable react/prefer-stateless-function */
 import React from 'react';
 import Calculator from './components/Calculator';
-import calculate from './logic/calculate';
 
 export default class App extends React.Component {
```

Comment:
- [OPTIONAL] Following ES6 syntax, you could implement it to your other components too. ♻️

Reply:
> 

What do you mean, please?

The export default ? :thinking: on line 8 or :thinking: ?

Reply:
Actioned :heavy_check_mark: